website/code_issues.qmd

```diff
@@ -80,8 +80,55 @@ In your examples, vignettes, demos and tests setting a seed is not only allowed
 
 -----------------------------------------------------------
 
-# Issue 3
+# Using print()/cat()
 
 ## Problem
 
+You are using functions, like print() or cat(), to print unsuppressable output to the console.
```

Comment:
I would use code syntax when referencing the functions:

```suggestion
You are using functions, like `print()` or `cat()`, to print unsuppressable output to the console.
```

Reply:
👍